sqlitedict.py

```diff
@@ -378,7 +378,10 @@ def __init__(self, filename, autocommit, journal_mode, timeout, outer_stack=True
         self.journal_mode = journal_mode
         # use request queue of unlimited size
         self.reqs = Queue()
-        self.setDaemon(True)  # python2.5-compatible
+        if sys.version_info.major == 3:
+            self.daemon = True  # python 3.x compatible
+        else:
+            self.setDaemon(True)  # python2.5-compatible
```

Comment:
We don't support Py2 anymore, so this is unnecessary.

Reply:
I've updated this now. Based on the comment in the original in the previous version (`# python2.5-compatible`) I thought that backwards compatibility with 2.5 was needed.